### DISCUSSION TITLE
Conditional Access Question

### DISCUSSION THREAD

**Discussion by mnagatos101** [azure-ad-b2c/samples#431] [Q&A]
According to the sample, it looks like objectId/User/Id is required to evaluate conditional access.  This means that a login attempt has to precede it, yes?

However, if I want to block access by location then I want to go straight to the block page without displaying the login page.  

How do I get the conditional access evaluation without a login attempt?  Is it even possible?

Please advise.

**Comment by piquet-h** [comment]
Conditional access provides for a different use case than the one you're solving for. It sounds like you want to deny everything for that location. 

Conditional access is about denying only certain sign-ins that meet certain criteria. It's about reducing risk of an insecure login rather than denying based on location.
